common/source/docs/common-camera-runcam.rst

```diff
@@ -0,0 +1,64 @@
+
+* In order to recognize the RunCam serial protocol on the UART, set the UART serial protocol to 26. So for instance for serial port 5, set :ref:`SERIAL5_PROTOCOL <SERIAL5_PROTOCOL>` 26 (= RunCam protocol). Adjust the serial port's protocol parameter to 26, for the serial port that your RunCam's serial wires are connected to, as required. The correlation between serial port numbering and UART physical ports for your autopilot should be documented in its description page linked :ref:`here <common-autopilots>`.
+  When the camera is connected correctly and the serial port configured correctly you will see a message similar to the following at boot: ``APM: RunCam initialized, features 0x0077, 2-key OSD`` otherwise you will see ``APM: RunCam device not found``.
+  The camera must be powered and fully booted in order to accept commands. This can take some time and the delay between the flight controller booting and the camera being probed can be controlled through :ref:`CAM_RC_BT_DELAY <CAM_RC_BT_DELAY>`. The camera will be probed repeatedly until :ref:`CAM_RC_BT_DELAY <CAM_RC_BT_DELAY>` elapses, but once that has expired the camera will not be probed again.
+
+* Some split-style cameras support three modes - video standby, video start and picture mode. It is not possible to tell from the RunCam device protocol whether this is the type of camera in use and so this must be configured. Set :ref:`CAM_RC_DEVICE_TYPE <CAM_RC_DEVICE_TYPE>` 2. All other cameras should be fine with the default.
+
+* In order to control start and stop of video recording from a radio transmitter set :ref:`RC8_OPTION <RC8_OPTION>` = 78 to enable triggering from transmitter switch 8. Adjust the switch number for triggering on different switches. If the switch is in the low position then video recording is stopped, if in the high position then video recording is started.
+
+* In order to control entering the OSD menu set :ref:`RC8_OPTION <RC8_OPTION>` = 79. Adjust the switch number for triggering the OSD menu on different switches. If the switch is in the low position then the OSD menu is exited, if in the high position then the OSD menu is entered. The OSD menu can also be entered by default through rolling hard right while disarmed.
+
+OSD Menu Control
+================
+
+A big part of the RunCam driver is support for the OSD menu. The OSD menu allows various properties of a connected camera to be controlled via RC stick gestures. The easiest way to enter the OSD menu is to push the stick for Roll hard right. This gesture is enabled by default but can be disabled by setting :ref:`CAM_RC_CONTROL <CAM_RC_CONTROL>`. Once in the menu, the Pitch stick will allow you to navigated between menu items and Roll right will usually allow you to select items or navigate into sub-menus.
```

Comment:
repeat that hard right is only during disarm to enter

Reply:
Added